app/assets/scripts/components/project/sec-panel/class-analytics-chart.js

```diff
@@ -139,34 +133,30 @@ function ClassAnalyticsChart(props) {
         />
       </ChartContainer>
       <Summary>
-        {bounds && (
-          <ClassItem bounds={bounds}>
-            <p> </p>
-            <Prose size='small'>CLASS NAME</Prose>
-            <Prose size='small' className='percent'>
-              AREA KM2
-            </Prose>
-            <Prose size='small' className='percent'>
-              %
-            </Prose>
-          </ClassItem>
-        )}
+        <ClassItem>
+          <p> </p>
+          <Prose size='small'>CLASS NAME</Prose>
+          <Prose size='small' className='percent'>
+            AREA KM2
+          </Prose>
+          <Prose size='small' className='percent'>
+            %
+          </Prose>
+        </ClassItem>
         {Object.values(checkpoint.classes).map(
           (c, i) =>
             checkpoint.analytics[i] && (
-              <ClassItem key={c.name} bounds={bounds}>
+              <ClassItem key={c.name}>
                 <StyledTooltip id={c.name} place='bottom' effect='float'>
                   {c.name}
                 </StyledTooltip>
                 <Icon color={c.color} data-tip data-for={c.name} />
                 <Prose size='small' data-tip data-for={c.name}>
                   {c.name}
                 </Prose>
-                {bounds && (
-                  <Prose size='small' className='percent'>
-                    {landArea(checkpoint.analytics[i][metric])}
-                  </Prose>
-                )}
+                <Prose size='small' className='percent'>
```

Comment:
keep conditional check - only need this column of data on the class distribution chart, but the component is also used for class F1 scores